### ISSUE TITLE
editorjs-undo not working with default editorjs configuration since 0.1.7

### ISSUE THREAD

**Issue by jrochette** [kommitters/editorjs-undo#49]
Hi 👋 

I update this morning to 0.1.7 and editorjs-undo stopped working. This is because my editorjs was configured with the default value for the `holder` param, which is `'editorjs'` (https://editorjs.io/getting-started#configuration).  It results in the following errors when instantiating Undo

```
bundle.js:280 Uncaught (in promise) TypeError: e.addEventListener is not a function
    at t.value (bundle.js:280)
    at new t (bundle.js:160)
    at onReady (fs_builder_controller.js:116)
    at editor.js:13098
 bundle.js:205 Uncaught (in promise) TypeError: Cannot read property 'undefined' of undefined
    at t.value (bundle.js:205)
    at bundle.js:199
 ```
 
Changing the configuration value of `holder` to `document.querySelector('#editorjs')` fixes the issue. I think editorjs-undo should support both a String or a HTMLElement value for the `holder` param, as this is what is supported by editorjs (see https://github.com/codex-team/editor.js/blob/master/types/configs/editor-config.d.ts#L16).

Let me know what you think. If you agree with my suggestion, I can go ahead and write a PR for this.

Cheers!

**Comment by juanhurtado10** [comment]
Hey @jrochette 

Totally agree with you, We've tested using a `String` as a holder element, and the error you mentioned above was raised. Please, go ahead and submit your PR :). 

Thanks for contributing 🙌🏻

**Comment by juanhurtado10** [comment]
@jrochette FYI, this was already fixed in v0.2.0.

Best
Juan
